controls/def.cf

```diff
@@ -161,6 +161,11 @@ bundle common def
         comment => "This variable defines the list of classes that should
                     trigger exec_command if any of them are defined.";
 
+      # Users allowed on sockets
+      # Set the users allowed by default, if not already set (e.g. via augments)
+      "control_executor_runagent_socket_allow_users"
+        slist => { "cfapache" },
```

Comment:
cfapache user only exists for enterprise hubs. There is no class guard, so this will fail everywhere else.

Reply:
I can restrict this variable to be set only when the default user actually exists, then I can define a class, e.g. `have_control_executor_runagent_socket_allow_users` if `def.control_executor_runagent_socket_allow_users` is defined and use `have_control_executor_runagent_socket_allow_users::` to restrict setting the list of users in the control body to when we have a list of users defined.

Reply:
I think I would prefer if it was based on enterprise and hub classes, rather than if user exists. But I guess it doesn't matter too much.

Reply:
Actually I don't know what I prefer, no strong opinion.

Reply:
alright, i switched to using classes, squashed